www/docs/providers/zoho.md

```diff
@@ -0,0 +1,26 @@
+---

```

Comment:
So there seems to be a very strange issue here. Our script that should generate some of the docs reads these .md files with some RegEx to extract the title and id. Since this file was created on Windows, newline sequences are created as CRLF, in contrast to what our regex reads (LF).

I kindly ask you to commit this file with LF formatting, so our build can succeed:

![image](https://user-images.githubusercontent.com/18369201/111231924-5f247b80-85ea-11eb-98c3-25aed478e743.png)


We should probably improve the script here https://github.com/nextauthjs/next-auth/blob/main/www/scripts/generate-providers.js to not to trip up in the future.

Reply:
Thanks for the quick review. I was wondering why the build failed 🙂

I've committed this change and it appears to be passing.